xbmc/platform/win10/network/NetworkWin10.cpp

```diff
@@ -55,6 +55,11 @@ typedef struct icmp_echo_reply {
 #endif //! IP_STATUS_BASE
 #include <Icmpapi.h>
 
+namespace
+{
+constexpr auto MAC_LENGTH = 6; // fixed MAC length used in CNetworkInterface
```

Comment:
please set variable type intead use `auto` on `constexpr` is more easy to read

Reply:
Done